src/components/CreateProject/ProjectForm/CreateProjectForm.tsx

```diff
@@ -22,10 +24,42 @@ const FormSchema = yup
   })
   .required();
 
+// NOTE: the spread is to avoid the type error with notification props not accepting data attribute
+const notifySuccessCreation = (name: string) =>
+  showNotification({
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    ...{ 'data-test': 'notification-success' },
+    color: 'successDefault',
+    title: messages.projects.create,
+    message: `پروژه با نام ${name} با موفقیت ساخته شد`,
+  });
+
+const notifyFailedCreation = (name: string) =>
+  showNotification({
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    ...{ 'data-test': 'notification-fail' },
+    color: 'errorDefault',
+    title: messages.projects.create,
+    message: `مشکلی در مرحله ایجاد پروژه ای بانام ${name} به وجود آمده است. لطفا دوباره تلاش کنید`,
+  });
+
 export const CreateProjectForm = ({ dismiss }: Props) => {
-  const onSubmit = React.useCallback(({ name, description }: FormSchema) => {
-    console.log('name:', name, 'description:', description);
-  }, []);
+  const [createProject] = useCreateProjectMutation();
```

Comment:
please connect this mutation's loading state to the submit button's loading properly